proto/dwtools/abase/l1.test/Str.test.s

```diff
@@ -8835,6 +8836,80 @@ ghij
 
 //
 
+function strLinesNearestReport( test )
+{
+  let got, expected
+  test.case = 'charsRange[0] one letter';
+  got = _.strLinesNearestReport({ src : 'function add( x,y ) { return x + y }', charsRangeLeft : [ 1, 7 ], gray : 1, numberOfLines : 1 });
+  // console.log( got );
```

Comment:
Please, remove lines `console.log` when test routine will be finished. 
The output from test object is needed only for debugging.